.github/workflows/docusaurus-swizzled-warning.yml

```diff
@@ -0,0 +1,22 @@
+name: Docusaurus swizzled component changes warning
+on:
+  pull_request_target:
```

Comment:
This workflow doesn't actually need to run on `pull_request_target`. It can just run on `pull_request`. I'm aware this is a copy & paste from close-bot, which probably could have also just used `pull_request`.

Reply:
Oh yea, my bad, didn't notice this when I started with close-bot as a ref for the file.
Fixed with 640567d89defc418918a8e2614289b971dc9f7b6